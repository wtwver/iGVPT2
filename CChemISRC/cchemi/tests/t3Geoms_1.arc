

                     SUMMARY OF PM6 CALCULATION, Site No: 1079

                                                       MOPAC2012 (Version: 14.142L)
                                                       Tue Mar  3 18:12:48 2015
                                                       No. of days left =  80

           Empirical Formula: H2 O  =     3 atoms

PM6 CHARGE=0 Singlet

 Mopac file generated by CChemI


     GEOMETRY OPTIMISED USING EIGENVECTOR FOLLOWING (EF).     
     SCF FIELD WAS ACHIEVED                                   

          HEAT OF FORMATION       =        -54.30617 KCAL/MOL =    -227.21699 KJ/MOL
          TOTAL ENERGY            =       -319.07752 EV
          ELECTRONIC ENERGY       =       -458.35418 EV
          CORE-CORE REPULSION     =        139.27666 EV
          GRADIENT NORM           =          0.55417
          DIPOLE                  =          2.06890 DEBYE    POINT GROUP:       C2v 
          NO. OF FILLED LEVELS    =          4
          IONIZATION POTENTIAL    =         11.903147 EV
          HOMO LUMO ENERGIES (EV) =        -11.903  4.064
          MOLECULAR WEIGHT        =         18.015
          COSMO AREA              =         42.45 SQUARE ANGSTROMS
          COSMO VOLUME            =         25.16 CUBIC ANGSTROMS

          MOLECULAR DIMENSIONS (Angstroms)

            Atom       Atom       Distance
            H     2    H     1     1.53182
            O     3    H     2     0.56096
            H     1    H     2     0.00000
          SCF CALCULATIONS        =          5
          WALL-CLOCK TIME         =          0.055 SECONDS
          COMPUTATION TIME        =          0.016 SECONDS


          FINAL GEOMETRY OBTAINED
* ===============================
* Input file for Mopac
* MM/SE Energy(kCal/mol) =0.000000
* ===============================
PM6 CHARGE=0 Singlet

 Mopac file generated by CChemI
  H     0.93223430 +1  -0.31572630 +1  -0.28649777 +1
  H     0.40770242 +1   1.06944976 +1  -0.67715816 +1
  O     0.53733372 +1   0.47817352 +1   0.05372952 +1
 
