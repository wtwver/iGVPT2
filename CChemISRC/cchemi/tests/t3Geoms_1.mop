* ===============================
* Input file for Mopac
* MM/SE Energy(kCal/mol) =0.000000
* ===============================
PM6 CHARGE=0 Singlet

Mopac file generated by CChemI
 H 0.932968 1 -0.310415 1 -0.304422 1
 H 0.414693 1 1.058467 1 -0.691006 1
 O 0.535395 1 0.479597 1 0.061704 1
