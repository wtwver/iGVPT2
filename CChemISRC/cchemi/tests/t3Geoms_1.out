 *******************************************************************************
 ** Site#: 1079         For non-commercial use only     Version 14.142L 64BITS**
 *******************************************************************************
 ** Cite this work as:                                                        **
 **                                                                           **
 ** (a) MOPAC2012, James J. P. Stewart, Stewart Computational Chemistry,      **
 **  Version 14.142L web: HTTP://OpenMOPAC.net         Days remaining:  80    **
 **                                                                           **
 **                                 AND                                       **
 **                                                                           **
 ** (b) Maia, J.D.C., et al., J. Chem. Theory Comput. 2012, 8, 3072-3081.     **
 **                DOI: 10.1021/ct3004645.                                    **
 *******************************************************************************
 **                                                                           **
 **                                MOPAC2012                                  **
 **                                                                           **
 *******************************************************************************

                            PM6 CALCULATION RESULTS


 *******************************************************************************
 *  CALCULATION DONE:                                Tue Mar  3 18:12:48 2015  *
 *  PM6      - The PM6 Hamiltonian to be used
 *  SINGLET  - SPIN STATE DEFINED AS A SINGLET
 *
 *
 *
 *                 CHARGE ON SYSTEM =     0
 *
 *
 *
 *  T=       - A TIME OF 172800.0 SECONDS REQUESTED
 *  DUMP=N   - RESTART FILE WRITTEN EVERY  7200.000 SECONDS
 *******************************************************************************
PM6 CHARGE=0 Singlet

 Mopac file generated by CChemI
   ATOM   CHEMICAL          X               Y               Z
  NUMBER   SYMBOL      (ANGSTROMS)     (ANGSTROMS)     (ANGSTROMS)
 
     1       H          0.93296800  *  -0.31041500  *  -0.30442200  *
     2       H          0.41469300  *   1.05846700  *  -0.69100600  *
     3       O          0.53539500  *   0.47959700  *   0.06170400  *
 


          CARTESIAN COORDINATES 

    NO.       ATOM         X         Y         Z

     1         H        0.9330   -0.3104   -0.3044
     2         H        0.4147    1.0585   -0.6910
     3         O        0.5354    0.4796    0.0617

     ===================== shared-memory parallel CPU Section ======================
     **                                                                           **
     **  MOPAC calculations will run in shared-memory parallel CPUs               **
     **                                                                           **
     **  Number of Threads =  8                                                   **
     **                                                                           **
     **                                                                           **
     **  For information on this work, see:                                       **
     **                                                                           **
     **  Maia, J.D.C., et al., J. Chem. Theory Comput. 2012, 8, 3072-3081.        **
     **        DOI: 10.1021/ct3004645.                                            **
     **                                                                           **
     ===================== shared-memory parallel CPU Section ======================
      


 General Reference for PM6:
 "Optimization of Parameters for Semiempirical Methods V: Modification of NDDO Approximations
 and Application to 70 Elements", J. J. P. Stewart, J. Mol. Mod., 13, 1173-1213 (2007)
 URL: http://www.springerlink.com/content/ar33482301010477/fulltext.pdf

           Empirical Formula: H2 O  =     3 atoms



      MOLECULAR POINT GROUP   :   C2v 


 SINGLET STATE CALCULATION


      RHF CALCULATION, NO. OF DOUBLY OCCUPIED LEVELS =    4

      Geometry optimization using EF

          DIAGONAL MATRIX USED AS START HESSIAN

 CYCLE:     1 TIME:   0.008 TIME LEFT:  2.00D  GRAD.:     8.847 HEAT: -54.20450
 CYCLE:     2 TIME:   0.000 TIME LEFT:  2.00D  GRAD.:     7.861 HEAT: -54.26992
 CYCLE:     3 TIME:   0.000 TIME LEFT:  2.00D  GRAD.:     3.817 HEAT: -54.29886
 CYCLE:     4 TIME:   0.000 TIME LEFT:  2.00D  GRAD.:     0.554 HEAT: -54.30616

     RMS GRADIENT =  0.55417  IS LESS THAN CUTOFF =  1.00000



 -------------------------------------------------------------------------------
PM6 CHARGE=0 Singlet

 Mopac file generated by CChemI


     GEOMETRY OPTIMISED USING EIGENVECTOR FOLLOWING (EF).     
     SCF FIELD WAS ACHIEVED                                   


                              PM6 CALCULATION
                                                       MOPAC2012 (Version: 14.142L)
                                                       Tue Mar  3 18:12:48 2015
                                                       No. of days left =  80




          FINAL HEAT OF FORMATION =        -54.30617 KCAL/MOL =    -227.21699 KJ/MOL


          TOTAL ENERGY            =       -319.07752 EV
          ELECTRONIC ENERGY       =       -458.35418 EV  POINT GROUP:     C2v 
          CORE-CORE REPULSION     =        139.27666 EV
          COSMO AREA              =         42.45 SQUARE ANGSTROMS
          COSMO VOLUME            =         25.16 CUBIC ANGSTROMS

          GRADIENT NORM           =          0.55417
          IONIZATION POTENTIAL    =         11.903147 EV
          HOMO LUMO ENERGIES (EV) =        -11.903  4.064
          NO. OF FILLED LEVELS    =          4
          MOLECULAR WEIGHT        =         18.015

          MOLECULAR DIMENSIONS (Angstroms)

            Atom       Atom       Distance
            H     2    H     1     1.53182
            O     3    H     2     0.56096
            H     1    H     2     0.00000


          SCF CALCULATIONS        =          5
 
          WALL-CLOCK TIME         =          0.055 SECONDS
          COMPUTATION TIME        =          0.016 SECONDS




   ATOM   CHEMICAL          X               Y               Z
  NUMBER   SYMBOL      (ANGSTROMS)     (ANGSTROMS)     (ANGSTROMS)
 
     1       H          0.93223430  *  -0.31572630  *  -0.28649777  *
     2       H          0.40770242  *   1.06944976  *  -0.67715816  *
     3       O          0.53733372  *   0.47817352  *   0.05372952  *
                             CARTESIAN COORDINATES

   1    H        0.932234302    -0.315726298    -0.286497768
   2    H        0.407702416     1.069449756    -0.677158157
   3    O        0.537333719     0.478173519     0.053729518


           Empirical Formula: H2 O  =     3 atoms



      MOLECULAR POINT GROUP   :   C2v 


                  EIGENVALUES  
 -30.39424 -18.98130 -14.10974 -11.90315   4.06355   6.03721


              NET ATOMIC CHARGES AND DIPOLE CONTRIBUTIONS

  ATOM NO.   TYPE          CHARGE      No. of ELECS.   s-Pop       p-Pop       
    1          H           0.309482        0.6905     0.69052
    2          H           0.309325        0.6907     0.69067
    3          O          -0.618807        6.6188     1.76030     4.85851
 DIPOLE           X         Y         Z       TOTAL
 POINT-CHG.     0.394    -0.302    -1.592     1.667
 HYBRID         0.095    -0.072    -0.384     0.402
 SUM            0.489    -0.374    -1.975     2.069


          ATOMIC ORBITAL ELECTRON POPULATIONS

    Atom      s        px        py        pz   
    1  H   0.69052
    2  H   0.69067
    3  O   1.76030   1.89462   1.43026   1.53363



 TOTAL JOB TIME:             0.05 SECONDS

 == MOPAC DONE ==
